### PR TITLE
Linkage Checker Enforcer Rule Release 1.0.1 

### PR DESCRIPTION
**Pull request by suztomo**
The enforcer rule now works with artifacts having `${os.detect.classifier}` in their dependencies.

### CODE REVIEW

**Comment by suztomo** [comment]
@elharo PTAL. Added CHANGELOG.md.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Linkage Checker Enforcer Rule Change Log
+All notable changes to this project will be documented in this file.
```
Comment (elharo) [inline CHANGELOG.md]: this line isn't needed

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+## 1.0.1
+* The enforcer rule now works with artifacts having `${os.detect.classifier}` in their dependencies.
```
Comment (elharo) [inline CHANGELOG.md]: Can you elaborate on this a little, perhaps link to the definition of this property in the Maven docs?

Comment (suztomo) [reply CHANGELOG.md]: Added explanation with a link to os-maven-plugin that defines the variable.

File: CHANGELOG.md (hunk 1 above)
Comment (suztomo) [reply CHANGELOG.md]: Removed.

**Comment by suztomo** [comment]
It's been pushed. The added explanation is the next line of the comment.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Linkage Checker Enforcer Rule Change Log
+
+## 1.0.1
+* The enforcer rule now works with artifacts having `${os.detect.classifier}` in their dependencies.
+  When such artifacts were resolved without [os-maven-plugin](https://github.com/trustin/os-maven-plugin)
```
Comment (suztomo) [inline CHANGELOG.md]: This is added explanation.

Comment (elharo) [reply CHANGELOG.md]: How about simply, "The enforcer rule now recognizes the `${os.detect.classifier}` property defined by the [os-maven-plugin](https://github.com/trustin/os-maven-plugin)."

That said, I wonder if we have a much bigger issue here. The current rule is is just patching over this one special case. There are many more cases involving os-maven-plugin and potentially other plugins where we are not resolving dependencies in the same way Maven does.

Comment (suztomo) [reply CHANGELOG.md]: Updated the changelog. Yes, there may be other cases. So far, only os-maven-plugin. I'm thinking to handle them in an ad-hoc manner.

**Pull request merged**